### DISCUSSION TITLE
Backward Compatibility ke php 7.4

### DISCUSSION THREAD

**Discussion by ivanwilliammd** [ivanwilliammd/satusehat-integration#119] [Ideas]
@YogiPristiawan @SyaefulKai @yudistirasd 
Apakah sdh ada yang pernah cek untuk library ini mungkin bisa backward compatible ke PHP v7.0 / 7.4?

Terimakasih

**Comment by yogip07** [comment]
Jika ingin backward compatibility ke php v7, sepertinya untuk beberapa sintaks perlu diubah supaya bisa compatible ke PHP v7 pak

**Comment by ivanwilliammd** [comment]
Boleh coba diusulkan untuk syntax yg perlu dirubah untuk backward compatible ke php 7.4 / 8.0? 
Barusan saya sudah bump composer.json dengan relaksasi requirement, dan superficially tested normal... 

Apakah bisa dibantu untuk in-depth checking @YogiPristiawan ?
Trims

**Comment by ivanwilliammd** [comment]
UP : @YogiPristiawan 
Note : Boleh dicoba dengan versi tag terbaru v2.1.0 -- in line dengan main per thread comment ini ?

**Comment by yogip07** [comment]
Baik pak. setelah weekday yang sangat sibuk, weekend ini coba saya periksa ASAP

**Comment by yogip07** [comment]
Perlukah mengganti testing tools menjadi PHP Unit pak @ivanwilliammd ?

**Comment by ivanwilliammd** [comment]
Benar @YogiPristiawan , sudah saya update di v2.2.0
Terimakasih atas bantuannya
